tests/src/DependencyInjection/ContainerTest.php

```diff
@@ -385,6 +385,41 @@ public function test_initialized() {
     $this->assertTrue($this->container->initialized('late.service'), 'Late service is initialized after it was gotten.');
   }
 
+  /**
+   * Camelizes a string.
+   *
+   * @covers ::camelize
+   * @dataProvider underscoreCamelizeDataProvider
+   */
+  public function test_camelize($string_underscore, $string_camelize) {
+    $result = $this->container->camelize($string_underscore);
+    $this->assertEquals($string_camelize, $result, 'The strings are the same.');
+  }
+
+  /**
+   * Un-camelizes a string.
+   *
+   * @covers ::underscore
+   * @dataProvider underscoreCamelizeDataProvider
+   */
+  public function test_underscore($string_underscore, $string_camelize) {
+    $result = $this->container->underscore($string_camelize);
+    $this->assertEquals($string_underscore, $result, 'The strings are the same.');
```

Comment:
Lets remove the 'The strings are the same.' here - it does not give new information.